### ISSUE TITLE
Voice cloning is not working (Error processing custom voice file: history prompt not found)

### ISSUE THREAD

**Issue by agrogov** [jgravelle/GroqCasters#6]
I tried to reuse the sample custom voice from bark_voice_samples/female.wav and just got `Error processing custom voice file: history prompt not found`...
Is there any solution for this issue?

**Comment by jgravelle** [comment]
Yeah, sample voices are hard.  You have to make your own history files,
which is doable with the clone-able version of Bark:

https://github.com/serp-ai/bark-with-voice-clone ...is a fork that
supposedly helps make these *.npz files, but I blew an entire weekend with
zero results.

This thread looks promising:  https://github.com/suno-ai/bark/issues/382

If you figure it out, let me know...!


-jjg

**Comment by agrogov** [comment]
I was able to successfully clone the example female voice with [HuBERT](https://github.com/gitmylo/bark-voice-cloning-HuBERT-quantizer), here is my modified [notebook](https://colab.research.google.com/drive/1nOWcu2qmbxffqeRLAze9FMFbBV0UxEvk?usp=sharing).
Just put generated `female.npz` in `bark_voice_samples` and that's it.

Changes in your code:
```python
CUSTOM_MALE_VOICE_PATH = None
CUSTOM_FEMALE_VOICE_PATH = "bark_voice_samples/female.npz"

class GroqCasters:
    def __init__(self):
        try:
            self.groq = GroqProvider()
            self._setup_gpu()
            preload_models()
            self.custom_male_voice = CUSTOM_MALE_VOICE_PATH
            self.custom_female_voice = CUSTOM_FEMALE_VOICE_PATH
```
And remove `_create_voice_prompt` function, cos it's no longer needed.

**Comment by jgravelle** [comment]
Dude!  This is huge!

Спасибо, товарищ...!


-jjg

**Comment by agrogov** [comment]
No problem!

And one more thing - you need to resample example voice to be 24KHz before, cos your initial file is 44KHz.
